src/imageio/imageio_exr.cc

```diff
@@ -250,7 +250,7 @@ dt_imageio_retval_t dt_imageio_open_exr(dt_image_t *img, const char *filename, d
     cmsxyY2XYZ(&srcWhite, &srcWhite_xy);
 
     /* use Imf::Chromaticities definition */
-    const cmsCIExyY d65_xy = {0.3127f, 0.3290f, 1.0};
+    const cmsCIExyY d65_xy = {D65xyY.x, D65xyY.y, 1.0};
```

Comment:
These actually came from the OpenEXR header files (as default values specified there) on purpose...

Reply:
oops, i killed this pr by accident ...